[2024-09-12T02:21:04.218+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-09-12T02:21:04.294+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: our_first_dag_V6.third_task scheduled__2023-01-21T00:00:00+00:00 [queued]>
[2024-09-12T02:21:04.322+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: our_first_dag_V6.third_task scheduled__2023-01-21T00:00:00+00:00 [queued]>
[2024-09-12T02:21:04.323+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 6
[2024-09-12T02:21:04.564+0000] {taskinstance.py:2888} INFO - Executing <Task(BashOperator): third_task> on 2023-01-21 00:00:00+00:00
[2024-09-12T02:21:04.577+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=7687) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-09-12T02:21:04.580+0000] {standard_task_runner.py:72} INFO - Started process 7718 to run task
[2024-09-12T02:21:04.593+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_V6', 'third_task', 'scheduled__2023-01-21T00:00:00+00:00', '--job-id', '20067', '--raw', '--subdir', 'DAGS_FOLDER/our_first_dag.py', '--cfg-path', '/tmp/tmp4114aj0p']
[2024-09-12T02:21:04.597+0000] {standard_task_runner.py:105} INFO - Job 20067: Subtask third_task
[2024-09-12T02:21:04.823+0000] {task_command.py:467} INFO - Running <TaskInstance: our_first_dag_V6.third_task scheduled__2023-01-21T00:00:00+00:00 [running]> on host 5508ea9ebfaf
[2024-09-12T02:21:05.233+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='acap' AIRFLOW_CTX_DAG_ID='our_first_dag_V6' AIRFLOW_CTX_TASK_ID='third_task' AIRFLOW_CTX_EXECUTION_DATE='2023-01-21T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-01-21T00:00:00+00:00'
[2024-09-12T02:21:05.242+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-09-12T02:21:05.295+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-09-12T02:21:05.300+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'echo hey, this is the third task!']
[2024-09-12T02:21:05.317+0000] {subprocess.py:86} INFO - Output:
[2024-09-12T02:21:05.323+0000] {subprocess.py:93} INFO - hey, this is the third task!
[2024-09-12T02:21:05.333+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-09-12T02:21:05.485+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-09-12T02:21:05.489+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=our_first_dag_V6, task_id=third_task, run_id=scheduled__2023-01-21T00:00:00+00:00, execution_date=20230121T000000, start_date=20240912T022104, end_date=20240912T022105
[2024-09-12T02:21:05.741+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-09-12T02:21:05.796+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
